src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Represents a Task. Contains a Task constructor, two methods to mark and unmark tasks, toString() method as well as a isMark() method to check if Task is marked
+ */
+public class Task {
+    private boolean mark;
+    public String name;
+
+    /**
+     * Constructor
```

Comment:
Might be better to have a more specific description of a constructor instead of simply "Constructor".
Recommendation:
```
/**
* Creates a task object with the given parameter as the name of the task.
* @param name name of the task
*/

```